docs/reference/commandline/version.md

```diff
@@ -61,12 +120,24 @@ Server:
   GitCommit:        fec3683
 ```
 
+## Examples
+
+### <a name=format></a> Format the output (--format)
+
+The formatting option (`--format`) pretty-prints container output using a Go
```

Comment:
I'd maybe just link https://docs.docker.com/config/formatting/ instead of repeating the explanation. But I also would not expect flag description to be again in the examples section but maybe just provide two very different examples.